.github/workflows/scripts/ti_build/llvm.py

```diff
@@ -44,4 +48,8 @@ def setup_llvm() -> None:
 
     # We should use LLVM toolchains shipped with OS.
     # path_prepend('PATH', out / 'bin')
-    os.environ["LLVM_DIR"] = str(out)
+    if (u.system, u.machine) not in (("Linux", "arm64"), ("Linux", "aarch64")):
+        os.environ["LLVM_DIR"] = str(out)
+        # os.environ["LLVM_DIR"] = "/usr/lib/llvm-20/cmake"
+        # os.environ["CUDA_HOME"] = "/usr/local/cuda"
```

Comment:
Let's remove dead code please.

Reply:
i prefer not to remove it, this is possible to build LLVM with local llvm(instaled in your SO)

Reply:
I feel like this should be parametrized via CMAKE somehow, rather than by using commented out code in the python scripts. Whilst it might be obvious to you that one should navigate to llvm.py, and uncomment out these lines, I feel like this would not not be obvious to other people.